app/helpers/layout_helper.rb

```diff
@@ -91,7 +93,10 @@ def multiple_selects(f, attr, associations, selected_ids, options = {}, html_opt
       JSON.parse(html_options["data-useds"]).each do |disabled_value|
         hidden_fields += f.hidden_field(attr_ids, :multiple => true, :value => disabled_value, :id=>'' )
       end
-      hidden_fields + f.collection_select(attr_ids, associations.all.sort_by { |a| a.to_s },
+      unauthorized.each do |unauthorized_value|
+        hidden_fields += f.hidden_field(attr_ids, :multiple => true, :value => unauthorized_value, :id=>'' )
+      end
+      hidden_fields + f.collection_select(attr_ids, authorized.sort_by { |a| a.to_s },
```

Comment:
Just a sanity check (and maybe laziness on my part..) , why do the unauthorized fields have to be sent as well? I think what @thomasmckay asks for in the ticket is to make sure sending operations on one ID works instead of on the whole set.